src/ui/screens/DashboardScreen.tsx

```diff
@@ -54,6 +56,7 @@ export function DashboardScreen({navigation, route}: Props) {
   return (
     <Layout style={styles.container}>
       <MainAppBar navigation={navigation} route={route} options={appBarOptions} />
+      <NetworkConnectionError />
```

Comment:
Is the UX designed to only show this in the Dashboard screen?

Reply:
Either a banner or toast or anything else, this should be at the root of the application.

Reply:
> Is the UX designed to only show this in the Dashboard screen?

@franjs, this does popup even on other screens also.